src/historic-quote/historic-quote.service.ts

```diff
@@ -96,12 +117,16 @@ export class HistoricQuoteService implements OnModuleInit {
       // Process Ethereum token mappings for all deployments
       await this.seedAllEthereumMappedTokens();
 
-      await Promise.all([
-        await this.updateCoinMarketCapQuotes(),
-        await this.updateCodexQuotes(BlockchainType.Sei),
-        await this.updateCodexQuotes(BlockchainType.Celo),
-        // await this.updateCodexQuotes(BlockchainType.Base, BASE_NETWORK_ID),
-      ]);
+      try {
+        await Promise.all([
```

Comment:
Small: you might to consider allSettled as it will wait for all promises to complete, regardless of whether or not one rejects - unlike Promise.all which rejects immediately upon first rejected promise.